README.md

```diff
@@ -362,12 +367,109 @@ The `$SERVICE_BINDING_ROOT` environment variable **MUST NOT** be reset if it is
 
 If a `.spec.type` is set, the `type` entry in the binding `Secret` **MUST** be set to its value overriding any existing value.  If a `.spec.provider` is set, the `provider` entry in the binding `Secret` **MUST** be set to its value overriding any existing value.
 
+### Binding Status
+
 If the modification of the Application resource is completed successfully, the `Ready` condition status **MUST** be set to `True`.  If the modification of the Application resource is not completed successfully the `Ready` condition status **MUST NOT** be set to `True`.
 
 # Extensions
 
 Extensions are optional additions to the core specification as defined above.  Implementation and support of these specifications are not required in order for a platform to be considered compliant.  However, if the features addressed by these specifications are supported a platform **MUST** be in compliance with the specification that governs that feature.
 
+## Custom Projection
+
+There are scenarios where the Reconciler that processes the bindings of a Provisioned Service (hereinafter referred to as `Reconciler A`) is different than the Reconciler that will project those bindings into the Application (hereinafter referred to as `Reconciler B`). To coordinate the projection between Reconcilers A and B the `ServiceBinding` CR author **MUST** set the `projection.service.binding` annotation to `Custom`.  
+
+Reconciler A reacts to this annotation by creating a new `ServiceBindingProjection` CR which includes the necessary information for Reconciler B, who is watching for and takes ownership of new `ServiceBindingProjection` resources (see the [Custom Projection Definition](#custom-projection-definition) section below), to carry out the projection.  Reconciler A is responsible for updating `ServiceBindingProjection` upon corresponding changes to `ServiceBinding`.
+
+Reconciler B **MUST** set the `ServiceBindingProjection`'s `Ready` condition according to the rules set in [Binding Status](#binding-status), while Reconciler A **MUST** reflect `ServiceBindingProjection`'s `Ready` condition in a new `ProjectionReady` condition inside `ServiceBinding`.  The `Ready` condition of `ServiceBinding` **MUST NOT** be set to `True` if its `ProjectionReady` condition is not `True`.
+
+
+### Requesting Custom Projection Example Resource
+
+```yaml
+apiVersion: service.binding/v1alpha2
+kind: ServiceBinding
+metadata:
+  name: account-service
+  annotations:
+    projection.service.binding: "Custom"
```

Comment:
I had thought about that, but it would collide with the convention used in the [annotation extension](https://github.com/k8s-service-bindings/spec#annotation-examples), which serves a different purpose.

Reply:
I like that.  will update!